src/components/toaster/toast/toast.tsx

```diff
@@ -0,0 +1,42 @@
+import classNames from 'classnames';
+import { CloseIcon } from '../../icons';
+import { toast, Toast as ReactHotToastType, resolveValue } from 'react-hot-toast';
+
+import styles from './toast.module.scss';
+
+interface ToastProps {
+    toastData: ReactHotToastType;
+    style?: React.CSSProperties;
+    className?: string;
+}
+
+export const Toast = ({ toastData, style, className }: ToastProps) => {
+    const defaultAnimation = toastData.visible ? styles.defaultEnter : styles.defaultExit;
+
+    return (
+        <div
```

Comment:
Doesn't it supposed to be `<ToastBar>` from `react-hot-toast`?

Reply:
`ToastBar` from `react-hot-toast` consists of a lot of inline styles and complex animation. 
1. I don't want to override them all.
2. If I override default styles with new inline styles, I cannot make them responsive.
3. User cannot edit  `ToastBar` with Style Panel in Codux. `ToastBar` doesn't have `className` prop.

Reply:
Inline styles contain only positioning and animation. And it accepts className `<ToastBar toast={{ className }} /> ` but not directly.
If your approach works and it is better for a Codux user, then okay